src/java.base/share/classes/java/util/TimeZone.java

```diff
@@ -63,9 +63,9 @@
  * along with a time zone ID. For instance, the time zone ID for the
  * U.S. Pacific Time zone is "America/Los_Angeles". So, you can get a
  * U.S. Pacific Time {@code TimeZone} object with:
- * <blockquote><pre>
```

Comment:
@naotoj \<blockquote> actually adds indentation for the contents contained within. So I will be reverting the removal of those tags to prevent visual change.